integrated_channels/canvas/client.py

```diff
@@ -78,22 +91,62 @@ def course_update_endpoint(canvas_base_url, course_id):
         )
 
     def create_content_metadata(self, serialized_data):
+        """
+        Creates a course in Canvas.
+        If course is not found ,easy!  create it as usual
+        If course found, it will be one of these workflow_states per doc /the current state
```

Comment:
`If course found, it will be one of these workflow_states per doc /the current state of the course one of 'unpublished', 'available', 'completed', or 'deleted'`

I'd maybe reword to something like
 
> If course is found, it will have one of the follow workflow_states: 
>                 `available`: issue an update with latest field values
>                 `completed`: this happens if a course has been concluded.
>                 ...
> for information of Canvas workflow_states see:
> https://canvas.instructure.com/doc/api/courses.html#method.courses.update